packages/notifier/README.md

```diff
@@ -1,6 +1,192 @@
 # Notifier
 
-Produces a notifier, which allows a service to notify clients of
-state changes using a stream of update promises.
+This package provides two similar abstractions for producing and consuming
+asyncronous sequences of values, the *NotifierKit* and the *SubscriptionKit*.
+Both enable a service to notify clients of state changes.
 
-For details, see the [documentation](https://agoric.com/documentation/distributed-programming.html#notifiers) .
+For user documentation on the NotifierKit, see the
+[documentation](https://agoric.com/documentation/distributed-programming.html#notifiers).
+The following is a more precise description of the semantics of the
+NotifierKit, the SubscriptionKit, and their distributed system properties.
+
+
+# Distributed Asynchronous Iteration
+
+JavaScript defines `AsyncGenerator`s, `AsyncIterable`s, and `AsyncIterator`s.
+Each of these manipulate an *async iteration*, an abstract sequence of
+values with the following structure:
+   * Any number of *non-final values* in a fully ordered sequence to be revealed
+     asynchronously over time. These may continue indefinitely or may be
+     terminated in one of two ways:
+   * The async iteration may *finish* successfully, reporting a final
+     *completion value*, or
+   * The async iteration may *fail*, reporting an alleged *reason* for its
+     failure. The reason is typically an error object, and should be, but may
+     in fact be any JavaScript value.
+
+The call `makeNotifierKit()` makes an `{updater, notifier}` pair.
+The call `makeSupscriptionKit()` for makes a similar
```

Comment:
Accepted suggestion. This was the first time I accepted one using the Github button for doing so. I didn't think to change the suggested text because I didn't think about it. Hence the failure of the conventional-commits check.